### PR TITLE
Adding binary search in python

### PR DESCRIPTION
**Pull request by minchi19**
Adding binary search in python with readme.md file

### CODE REVIEW

**Comment by minchi19** [comment]
Also added 0 - 1 knapsack problem in c++
Please review and merge

File: search-algos/binary_search/binary search.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+print("Enter the number of elements you want to search from")
```
Comment (khanchi97) [inline search-algos/binary_search/binary search.py]: This implementation is not very good,please take a look at binary_search.c code and run that code and see the difference.

**Review by khanchi97 — CHANGES_REQUESTED** [review]

File: search-algos/binary_search/binary search.py (hunk 1 above)
Comment (minchi19) [reply search-algos/binary_search/binary search.py]: Ok will try to give more clean code.
Please review 0-1 knapsack also

**Comment by minchi19** [comment]
I have updated binary search program

**Comment by khanchi97** [comment]
@minchi19 It's not working fine.Please try different case and try to make it good.

**Pull request closed without merging**